sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -524,6 +524,144 @@ public virtual RecognizeBusinessCardsOperation StartRecognizeBusinessCardsFromUr
 
         #endregion
 
+        #region Invoices
+
+        /// <summary>
+        /// Recognizes values from one or more invoices.
+        /// <para>See <a href="https://aka.ms/formrecognizer/businesscardfields"/> for a list of available fields on an invoice.</para>
```

Comment:
I created this: https://aka.ms/formrecognizer/invoicefields

Points to FR docs at the moment. I have a note to update it to the actual fields once that's published.